.github/actions/check-disk-space/action.yml

```diff
@@ -0,0 +1,68 @@
+name: Check remaining disk space
```

Comment:
Nit: Could you change this name and the file name to include the word "clean"?  That'll help make the purpose of the action clearer

And unless this works on all platforms, consider including the word "mac" as well